gems/pending/spec/util/miq_process_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'util/miq-process'
```

Comment:
As much as I hate it, should this file be named `miq-process_spec.rb` to match the other file name?